sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/PartitionManager.cs

```diff
@@ -36,7 +36,7 @@ public abstract class PartitionManager
         ///
         /// <param name="partitionOwnership">An enumerable containing all the ownership to claim.</param>
         ///
-        /// <returns>An enumerable containing all the existing ownership for the associated Event Hub and consumer group.</returns>
+        /// <returns>An enumerable containing the successfully claimed ownership.</returns>
```

Comment:
While technically correct, this seems to read a bit weird.  Maybe add "instances" to the end?